backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/graphicpromotion/internal/BpkGraphicPromoImpl.kt

```diff
@@ -247,35 +249,50 @@ private fun ForegroundContent(
     }
 }
 
-private object InteractiveBackgroundIndication : Indication {
+private object InteractiveBackgroundIndicationNodeFactory : IndicationNodeFactory {
 
-    @Composable
-    override fun rememberUpdatedInstance(interactionSource: InteractionSource): IndicationInstance {
-        val isPressed by interactionSource.collectIsPressedAsState()
+    override fun create(interactionSource: InteractionSource): DelegatableNode {
+        return InteractiveBackgroundIndicationNode(interactionSource)
+    }
 
-        val scale by animateFloatAsState(
-            targetValue = if (isPressed) 1.05f else 1.0f,
-            animationSpec = interactiveBackgroundAnimationSpec,
-            label = "background scale",
-        )
+    override fun hashCode(): Int = -1
```

Comment:
the example had that?! i am not sure when this is actually used and has an impact. but it was working as expected 🤷‍♀️